### PR TITLE
Test on Python 3.6 and 3.7

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

**Comment by kryskool** [comment]
Hi @jd 

it's ok for Python 3.6, but for Python 3.7, we cannot add it with this syntax, some tickets are open in travis bug tracker

2 solutions are provides:

- Use [TOXENV matrix](https://github.com/deadsnakes/travis-ci-python3.7-example/blob/master/.travis.yml)
- Use simple [matrix configuration](https://github.com/simplejson/simplejson/blob/master/.travis.yml)

Regards,

Some references:

- https://github.com/travis-ci/travis-ci/issues/9069
- https://github.com/travis-ci/travis-ci/issues/9815

**Comment by jd** [comment]
Thanks @kryskool! I'll look into those.

**Comment by jd** [comment]
Pull-request updated, HEAD is now 1cdfebbde52a6ab99240cbfec9bdb69bcc328934

**Comment by jd** [comment]
Pull-request updated, HEAD is now 2ba360e1388ea2bbacdc081148407f207f1d91e6

**Comment by jd** [comment]
Pull-request updated, HEAD is now e63c7289aa734627eb60bf574837156d8747fe91

**Comment by jd** [comment]
Pull-request updated, HEAD is now f521d18891ee54c30d0a66fd98ac3c834c874351

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 sudo: false
 language: python
-python: 3.5
+matrix:
+  include:
+    - python: 3.5
+    - python: 3.6
+    - python: 3.7
+      dist: xenial
+      sudo: true
```
Comment (sileht) [inline .travis.yml]: is sudo really required ?

Comment (jd) [reply .travis.yml]: Yes, that's what I read on the Internet.

Comment (kryskool) [reply .travis.yml]: Yes i confirm, sudo is required on xenial

**Pull request merged**